src/openassetio-core/include/openassetio/log/ConsoleLogger.hpp

```diff
@@ -0,0 +1,36 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2022 The Foundry Visionmongers Ltd
+
+#include <openassetio/export.h>
+#include <openassetio/log/LoggerInterface.hpp>
+
+#pragma once
+namespace openassetio {
+inline namespace OPENASSETIO_CORE_ABI_VERSION {
+namespace log {
+OPENASSETIO_DECLARE_PTR(ConsoleLogger)
+/**
+ * A logger that sends messages to the console (std:cerr).
+ */
+class OPENASSETIO_CORE_EXPORT ConsoleLogger final : public LoggerInterface {
+ public:
+  /**
+   * Creates a new instance of the ConsoleLogger
+   *
+   * @param colorOutput When true, messages will be colored based on
+   * their severity.
+   */
+  [[nodiscard]] ConsoleLoggerPtr static make(bool colorOutput = true);
+
+  /**
+   */
+  void log(Severity severity, const Str& message) override;
+
+ private:
+  explicit ConsoleLogger(bool colorOutput = true);
+
+  bool colorOutput_;
```

Comment:
[consider][minor] Super minor but I always try to use an `is` prefix for booleans, i.e. `isColorOutput`, so it's clear at a glance.  Maybe that's a bit old-school though.

Reply:
Renamed [here]()https://github.com/OpenAssetIO/OpenAssetIO/pull/552/commits/bffe11744668200772b6a459a4779b6eafc12c95).